lang/cs/Org.Apache.REEF.Examples.HelloREEF/HelloREEF.cs

```diff
@@ -116,7 +119,8 @@ private static IConfiguration GetRuntimeConfiguration(string name)
                         .Set(AzureBlockBlobFileSystemConfiguration.ConnectionString, connectionString)
                         .Build();
                 case AzureBatch:
-                    return AzureBatchRuntimeClientConfiguration.ConfigurationModule
+                    List<string> ports = new List<string> { "2000", "2001", "2002", "2003", "2004" };
+                    return AzureBatchRuntimeClientConfiguration.GetConfigurationModule(ports)
                         .Set(AzureBatchRuntimeClientConfiguration.AzureBatchAccountKey, @"##########################################")
```

Comment:
This code implicitly bind TcpPortprovider at runtime together with change in AzureBatchRuntimeClientConfiguration. Just want to check what TcpPortPrivder in client is used for.

Reply:
Removed SetTcpPortprovider.cs and TcpPortSet.cs and used AzureBatchPoolDriverPortsList in AzureBatchService class directly.

---
In reply to: [201496327](https://github.com/sharathmalladi/reef/pull/5#discussion_r201496327) [](ancestors = 201496327)